galasa-ui/src/components/test-runs/ArtifactsTab.tsx

```diff
@@ -201,18 +184,13 @@ export function ArtifactsTab({
         let currentNode: FolderNode = root;
         createFolderSegments(segments, currentNode, artifact);
       }
-
-      // Check for zos3270/terminals folder
-      if (segments[0] === 'zos3270' && segments[1] === 'terminals') {
-        // const folderExists = Object.values(currentNode.children).some(child => child.isFile === false);
-        console.log(`zos3270/terminals folder exists:`);
-        // You can add additional logic here if needed, e.g., highlight or mark this folder
-      } else {
-        console.log(`zos3270/terminals DOESN'T EXIST`);
-      }
     });
 
     setTreeData(root);
+
+    if (is3270ScreenEnabled) {
+      checkForZosTerminalFolderStructure(root, setZos3270TerminalFolderExists);
+    }
   }, [artifacts]);
```

Comment:
I think this useEffect will now complain that `setZos3270TerminalFolderExists` isn't in the dependency array. If so, can this be resolved?